src/genie/libs/parser/iosxr/show_isis.py

```diff
@@ -0,0 +1,172 @@
+"""
+show_isis.py
+
+IOSXR parsers for the following show commands:
+    * show isis adjacency
+    * show isis neighbors
+
+"""
+
+# Python
+import re
+from netaddr import IPAddress, IPNetwork
+
+# Metaparser
+from genie.metaparser import MetaParser
+from genie.metaparser.util.schemaengine import Schema, Any, Or, Optional
+from genie.libs.parser.utils.common import Common
+
+
+#==================================
+# Schema for 'show isis adjacency'
+#==================================
+class ShowIsisAdjacencySchema(MetaParser):
+    """Schema for show run isis adjacency"""
+        
+    schema = {
+        'isis': {
+            Optional(Any()): {
```

Comment:
Having all the keys as 'Optional' is safe but risky because it might hide parsing flaws (if device output is changed).